RxCocoa/iOS/WKWebView+Rx.swift

```diff
@@ -52,6 +52,18 @@ extension Reactive where Base: WKWebView {
                 )
             }
     }
+
+    public var didFailProvisionalNavigation: Observable<(WKNavigation, Error)> {
```

Comment:
Please add a corresponding test in WKWebView+Tests.swift

Reply:
@freak4pc 
Thank you for your feedback.It took some time, but I've committed the changes.
Added test code to WKWebView+RxTests.swift.